spec/delayed/backend/active_record_spec.rb

```diff
@@ -115,7 +115,7 @@
     end
   end
 
-  if ::ActiveRecord::VERSION::MAJOR < 4 || defined?(::ActiveRecord::MassAssignmentSecurity)
+  if ActiveRecord::VERSION::MAJOR < 4 || defined?(ActiveRecord::MassAssignmentSecurity)
```

Comment:
Resolve the Rubocop check: [`Style/RedundantConstantBase`](https://docs.rubocop.org/rubocop/cops_style.html#styleredundantconstantbase).